src/repository/resources/ipres.rs

```diff
@@ -1709,12 +1709,128 @@ impl AddressFamily {
 }
 
 
+//------------ Ipv4Block -----------------------------------------------------
+
+/// Contains an IPv4 address block.
+///
+/// This type is a thin wrapper around [`IpBlock`] ensuring that this is an
+/// IPv4 address block.
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct Ipv4Block(IpBlock);
+
+impl Ipv4Block {
+    /// Creates a new block covering all addresses.
+    pub fn all() -> Self {
+        Self(IpBlock::all())
+    }
+
+    /// Returns whether the block is prefix with address length zero.
```

Comment:
This is not per se an issue with this PR as it seems to be a copy paste from the existing `IpBlock` code, but shouldn't this say "is prefixed with" rather than "is prefix with" ?

Reply:
Not quite: it was supposed to say “is _a_ prefix with length zero.”